apps/stats-dapp/webpack.config.js

```diff
@@ -338,7 +338,7 @@ function createWebpackBase(env, mode = 'production') {
 
     // https://webpack.js.org/configuration/dev-server/
     devServer: {
-      port: 3001,
+      port: process.env.PORT_STATS_DAPP,
```

Comment:
Consider adding a fallback port here if the env is not provided.

Reply:
i've added fallback [here](https://github.com/webb-tools/webb-dapp/pull/848/commits/c1abc8a3b4574e07cf06d2c7e34b91370b27bd16#diff-4a94c74929bd38fab1d21ca563ac75cda56963e3bd9f0188f95275da4af6bb90R341)